src/Site.js

```diff
@@ -364,7 +365,7 @@ Site.prototype.collectAddressablePages = function () {
     globPages.concat(walkSync(this.rootPath, {
       directories: false,
       globs: [addressableGlob.glob],
-      ignore: [BOILERPLATE_FOLDER_NAME],
+      ignore: [BOILERPLATE_FOLDER_NAME, CONFIG_FOLDER_NAME],
```

Comment:
Can remove `BOILERPLATE_FOLDER_NAME` since it's a subdirectory of `CONFIG_FOLDER_NAME`?